BusyList/ITaskRepository.cs

```diff
@@ -4,7 +4,16 @@ namespace BusyList
 {
     public record AddTaskData(string Description);
 
-    public record TaskItem(int Id, string Description);
+    public record TaskItem(int Id, string Description)
+    {
+        public string Print()
+        {
+            string changedString = $"#{Id} with description: {Description}";
```

Comment:
You can just return this string from here instead of creating a variable for it. It's called "inlining" the return.